installer/checkSystem.php

```diff
@@ -62,6 +63,35 @@ function checkMemory() {
 	return $msg;
 }
 
+function isPhpCompatible() {

```

Comment:
Is it necessary to have another function ? I can see that it's initializing 
$systemValidator = new systemValidator(); in each functions 

What if keeping the $systemValidator in global space call whenever necessary ?